common/src/main/java/tachyon/util/io/BufferUtils.java

```diff
@@ -284,4 +284,8 @@ public static void writeBufferToFile(String path, byte[] buffer) throws IOExcept
     os.write(buffer);
     os.close();
   }
+
+  private BufferUtils() {
```

Comment:
``private BufferUtils() {} // prevent instantiation`` is fewer lines of code and more informative